pillar/dagster/mitx_enrollments.sls

```diff
@@ -9,16 +11,10 @@ dagster:
           config:
             s3_bucket: dagster-{{ environment }}
             s3_prefix: pipeline-storage/xpro_edx
-        results_dir:
-          config:
-            outputs_directory_date_format: '%Y%m%d'
-        healthchecks:
```

Comment:
Why are we removing the healthcheck?

Reply:
That was never actually set up for this pipeline so removing the unused config. Something to revisit soon though.